__tests__/app.test.js

```diff
@@ -150,10 +150,37 @@ describe('PATCH', () => {
         })
     });
 })
-})
-describe('ERROR', () => {
+
+describe('POST', () => {
+    describe('/api/articles/:article_id/comments', () => {
+        test('Status: 201, returns the added object', () => {
+            const newComment = {
+                username: "butter_bridge",
+                body: "What a wonderfully artistic vision",
+            };
+            const expected = {
+                comment: {
+                    comment_id: 19,
+                    body: "What a wonderfully artistic vision",
+                    author: "butter_bridge",
+                    article_id: 1,
+                    created_at: expect.any(String),
+                    votes: 0
+                }
+            };
+            return request(app)
+            .post('/api/articles/1/comments')
+            .send(newComment)
+            .expect(201)
+            .then((res) => {
+                expect(res.body).toEqual(expected)
```

Comment:
I would still utilise `expect.objectContaining` here to allow additional properties in future